src/app/[locale]/(routes)/mypositions/page.tsx

```diff
@@ -9,7 +11,7 @@ const page: FC<pageProps> = () => {
     <section className="my-4 xl:my-8 space-y-4 lg:space-y-8">
       <PageTitle title="My Positions" />
       <PositionTabs />
-      <PositionCards />
+      <PositionCards positionCardsData={positionCardsData} />
```

Comment:
This data comes from an inpirt.

I belive is going to be replace later on.
But the import could have being just done inside

Reply:
I just made it to be passed as prop, even if its coming from input we still need to pass the data to the component. the component is a reusable component